packages/manager/modules/manager-pci-common/src/api/hook/usePCIFeatureAvailability.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from '@tanstack/react-query';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { useApplicationName } from '@/hooks/useApplicationName/useApplicationName';
+
+type FeatureAvailabilityQueryParams = Partial<{
```

Comment:
If there is only one property the `Partial` looks useless

Reply:
It is to mark all properties partial since they are query params, today there is only but there could more tomorrow and other devs might not think of marking it as not required